_posts/2017-11-09-totw-49.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Tip of the Week #49: Argument-Dependent Lookup"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/49
+type: markdown
+order: "049"
+---
+
+Originally posted as totw/49 on 2013-07-14
+
+*"...whatever disappearing trail of its legalistic argle-bargle one chooses to
+follow..." --Antonin Scalia, [U.S. v Windsor dissenting opinion]
+(http://www.supremecourt.gov/opinions/12pdf/12-307_6j37.pdf)**
+
+## Overview
+
+A function call expression such as `func(a,b,c)`, in which the function is named
+without the `::` scope operator, is called unqualified. When C++ code refers to
+a function by an unqualified name, the compiler performs a search for a matching
+function declaration. What is surprising to some people (and different from
+other languages) is that in addition to the caller's lexical scope, the set of
+search scopes is augmented by namespaces associated with the function argument
+types. This additional lookup is called Argument-Dependent Lookup (ADL). It's
```

Comment:
Nit: italicize first use of new terms with acronyms

*Argument-Dependent Lookup* (ADL)

Reply:
Done.